NitroxServer/ConsoleCommands/AutosaveCommand.cs

```diff
@@ -9,15 +8,19 @@ namespace NitroxServer.ConsoleCommands
 {
     internal class AutoSaveCommand : Command
     {
-        public AutoSaveCommand() : base("autosave", Perms.ADMIN, "Toggles the map autosave")
+        private readonly ServerConfig serverConfig;
+
+        public AutoSaveCommand(ServerConfig serverConfig) : base("autosave", Perms.ADMIN, "Toggles the map autosave")
         {
             AddParameter(new TypeBoolean("on/off", true));
+
+            this.serverConfig = serverConfig;
         }
 
         protected override void Execute(CallArgs args)
         {
             bool toggle = args.Get<bool>(0);
-            ServerConfig serverConfig = NitroxConfig.Deserialize<ServerConfig>();
+
```

Comment:
Removing this will cause existing changes in config file to be overwritten. I think this is suprising behaviour for users so I recommend adding it back.